style.css

```diff
@@ -505,6 +513,60 @@ hr {
   flex-wrap: wrap;
 }
 
-.footer p {
+.footer {
+  background-color: #464646;
+}
+
+.footer p,
+a {
   margin: 0 0.25em;
+  color: #ffffff;
+  font-family: Al Nile;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+}
+.footer .credits {
+  margin: auto;
+  order: 2;
+}
+.footer navbar {
+  margin-right: auto;
+  margin-top: auto;
+  margin-bottom: auto;
+  flex-direction: row;
+  order: 3;
+}
+.footerLogo {
+  margin-left: auto;
+  margin-top: auto;
+  margin-bottom: auto;
+  position: relative;
+  order: 1;
+}
+
+.footer navbar span {
+  display: none;
+  color: #ffff;
+}
+
+@media screen and (max-width: 490px) {
```

Comment:
Why 490px?

Reply:
That's the point where the items don't fit in the same row.